### PR TITLE
Square/refunds

### PR DESCRIPTION
**Pull request by jacobrobertbaca**
# Description of change
added refunds stream and tests

# Manual QA steps
 - ran with records and made sure they came through correctly
 - added to tests and ran tests
 
# Risks
 - low
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/base.py [hunk 1]
```diff
@@ -65,16 +65,8 @@ def get_credentials():
...
-        return {
-            'items',
-            'categories',
-            'discounts',
-            'taxes',
-            'employees',
-            'locations',
-        }
+    def expected_check_streams(self):
+        return set(self.expected_metadata().keys()).difference(set())
```
Comment (luandy64) [inline tests/base.py]: This is a duplicate of #18

File: tests/base.py [hunk 2]
```diff
@@ -227,7 +224,7 @@ def modify_expected_datatypes(self, expected_records):
 
     def align_date_type(self, record, key, value):
         """datetime values must conform to ISO-8601 or they will be rejected by the gate"""
-        if isinstance(value, str) and key in ['updated_at']:
+        if isinstance(value, str) and key in ['updated_at', 'created_at']:# TODO: update to reflect replication keys
```
Comment (luandy64) [inline tests/base.py:232]: Should this TODO be done in this PR?

**Review by luandy64 — COMMENTED** [review]
I have the one comment about the `TODO` before I think this is good

File: tests/base.py (hunk 2 above)
Comment (jacobrobertbaca) [reply tests/base.py:232]: no, @kspeer825 asked me to put it there for him

File: tests/base.py (hunk 1 above)
Comment (jacobrobertbaca) [reply tests/base.py]: yeah #18 wasn't ready yet so i threw it in here too

**Review by luandy64 — APPROVED** [review]
Looks good to me, I wonder if a merge conflict will happen

**Pull request merged**